src/EnergyPlus/SingleDuct.cc

```diff
@@ -1791,8 +1791,11 @@ namespace SingleDuct {
 		//Do a check and make sure that the max and min available(control) flow is
 		//  between the physical max and min while operating.
 		SysInlet( SysNum ).AirMassFlowRateMaxAvail = min( Sys( SysNum ).AirMassFlowRateMax, Node( InletNode ).MassFlowRateMaxAvail );
-		SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
-
+		if ( Sys( SysNum ).SysType_Num == SingleDuctConstVolReheat ) { // AirTerminal:SingleDuct:ConstantVolume:Reheat
```

Comment:
For the OA units (or I guess any zone HVAC really but I haven't thought through those yet and whether they should ever be imbalanced), if the imbalance could somehow be communicated back to the OA mixer, then the extra air could be exhausted and problem solved because ResolveAirLoopFlowLimits looks at supply side outlet and demand side inlet. I'm also thinking that function assumes balanced flow, so if we balance one way or another that function won't bother us again.